src/bootstrap/src/bin/main.rs

```diff
@@ -210,15 +210,25 @@ fn check_version(config: &Config) -> Option<String> {
 // - `tracing`'s `#[instrument(..)]` macro will need to be gated like `#![cfg_attr(feature =
 //   "tracing", instrument(..))]`.
 #[cfg(feature = "tracing")]
-fn setup_tracing() {
+fn setup_tracing() -> impl Drop {
     use tracing_subscriber::EnvFilter;
     use tracing_subscriber::layer::SubscriberExt;
 
     let filter = EnvFilter::from_env("BOOTSTRAP_TRACING");
     // cf. <https://docs.rs/tracing-tree/latest/tracing_tree/struct.HierarchicalLayer.html>.
     let layer = tracing_tree::HierarchicalLayer::default().with_targets(true).with_indent_amount(2);
 
-    let registry = tracing_subscriber::registry().with(filter).with(layer);
+    let mut chrome_layer = tracing_chrome::ChromeLayerBuilder::new().include_args(true);
+
+    // Writes the Chrome profile to trace-<unix-timestamp>.json if enabled
```

Comment:
Question: does this respect bootstrap out dir? It probably should just dump the json to `builder.out`, right?

Reply:
Hmm, yes, in theory, but we currently execute a bunch of commands even before we create `builder` and the config :upside_down_face:

Reply:
Ah, of course, this is way too early. Could we maybe hijack the `BOOTSTRAP_PROFILE` env var to accept a *path* for the output json instead, or is that too hacky?

Reply:
We surely could, I was just thinking that it might be a bit more practical if you can just rerun the same command a bunch of times and let it generate a few files, and then you can inspect them at your own pace.